global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "1.0.0-preview2-003131"
+    "version": "2.1.403"
```

Comment:
consider deleting file

Reply:
current fallback behaviour should ensure any sdk 2.1 version is used, can keep until it gives problems